docs/index.rst

```diff
@@ -1,7 +1,7 @@
-SQLAlchemy-Continuum
+SQLAlchemy-History
```

Comment:
Have you done a find replace or a manual chage everywhere ?
It would be good to describe the process you followed in the github commit message so reviewers can understand and review th eprocess instea of checking every line rigorously